w09h03/test/pgdp/filter/CropTest.java

```diff
@@ -0,0 +1,63 @@
+package pgdp.filter;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.awt.image.BufferedImage;
+import java.util.function.Function;
+
+public class CropTest {
+  @Test
+  void testCrop_exactSize() {
```

Comment:
Method should be written in [lowerCamelCase](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) ([5.2.3 Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names)).

Reply:
Yep. My bad.